src/deva/utils/platform.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2024-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+import os
+import sys
+
+if sys.platform == "win32":
+    PLATFORM_ID = "windows"
+    PLATFORM_NAME = "Windows"
+    DEFAULT_SHELL = os.environ.get("SHELL", os.environ.get("COMSPEC", "cmd"))
```

Comment:
There is no logic yet but the shell will be based on the parent process and this will be used as a fallback. When entering a container or VM, the images will be known so the proper nice shell will be used but for unknown/custom images then this will be used as well.